frontend/lang/italiano.json

```
@@ -0,0 +1,311 @@
+    "t_cell_indirectly_disabled_in_file_explanation": "Questa cella è attualmente memorizzata nel file del notebook come <em>commento</em>, invece che come <em>codice</em>.<br />In questo modo, il codice della cella non verrà eseguito quando il notebook viene usato come script al di fuori di Pluto.<br />Una cella a monte sta <b>indirettamente</b> <em>disabilitando nel file</em> questa; abilita {{upstreamLink}} per influenzare questa cella.",
+    "t_language_picker_description": "Cambia la lingua dell'interfaccia",
+    "t_refresh_to_see_language_change_confirm": "Lingua cambiata! Vuoi aggiornare la pagina per vedere le modifiche?",
+
+    "t_anonymous_function_abbr": "Una (mini-)funzione che è definita senza la parola chiave 'function', ma usando -> o 'do'.",
+    "t_display_complete_type_information_of_this_function_call": "Mostra le informazioni complete dei tipi di questa chiamata di funzione",
+    "t_multiple_expressions_in_one_cell": "Espressioni multiple in una cella",
+    "t_how_would_you_like_to_fix_it": "Come vorresti risolvere questo problema?",
+    "t_split_this_cell_into_cells": "Dividi questa cella in {{count}} celle",
+    "t_wrap_all_code_in_a_begin_end_block": "Racchiudi tutto il codice in un blocco <em>begin ... end</em>.",
+    "t_error_message_from_package": "Messaggio di errore da {{package}}",
+    "t_multiple_definitions_for": "Definizioni multiple per {{symbols}}.",
+    "t_cyclic_references_among": "Riferimenti ciclici tra {{symbols}}.",
+    "t_combine_cells_begin_block": "Combina tutte le definizioni in una singola cella reattiva usando un blocco <code>begin ... end</code>.",
+    "t_another_cell_defining_xs_contains_errors": "Un'altra cella che definisce {{symbols}} contiene errori.",
+    "t_error_message": "Messaggio di errore",
+    "t_stack_trace": "Stack trace",
+    "t_header_list_of_syntax_errors": "Errori di sintassi",
+    "t_here_is_what_happened_the_most_recent_locations_are_first": "Ecco cosa è successo, le posizioni più recenti sono per prime:",
+    "t_show_stack_trace": "Mostra stack trace...",
+    "t_stack_frame_location": "da",
+    "t_stack_frame_this_cell": "Questa cella",
+    "t_stack_frame_other_cell": "Altra cella",
+    "t_stack_frame_line": "riga",
+    "t_show_more": "Mostra di più...",
+    "t_motivational_words_be_creative_and_write_as_many_as_you_want": [
+        "Non farti prendere dal panico!",
+        "Mantieni la calma, ce la puoi fare!",
+        "Ce la puoi fare!",
+        "Computer sciocco!",
+        "Questo è colpa del computer!",
+        "beep boop CRASH 🤖",
+        "computer cattivo, tu GRANDE!",
+        "Probabilmente non è colpa tua!",
+        "Prova a chiedere su Julia Discourse!",
+        "uhmmmmmm??!",
+        "Forse è ora di una pausa? ☕️",
+        "Tutto andrà bene!",
```

That sounds too literal of a translation
```suggestion
        "Andrà tutto bene!",
```
feels slightly more familiar to me.